learning_table_tennis_from_scratch/models.py

```diff
@@ -57,17 +65,37 @@ def run_openai_baselines(
 
     tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.ERROR)
 
+    from baselines import logger
     from stable_baselines.common import make_vec_env
 
+    class LoggerWrapper:
+        """Wrapper for baselines.logger so it has same methods as stable_baselines3."""
+
+        def __init__(self, logger):
+            self.logger
+
+        def record(self, key, value):
+            self.logger.logkv(key, value)
+
+        def dump(self):
+            self.logger.dumpkvs()
+
+    ppo_config = OpenAIPPOConfig.from_json(ppo_config_file)
+
+    if ppo_config["log_tensorboard"]:
+        tensorboard_logger = LoggerWrapper(logger)
+    else:
+        tensorboard_logger = None
+    del ppo_config["log_tensorboard"]
```

Comment:
I guess this is to ensure the configuration entry is not read twice ? (neat !)

Reply:
I copied this pattern from the other parameters below.  I think the purpose is to remove it before doing `learn(..., **ppo_config)` below where it would likely cause an error.